feat(aps): utm how to guides (#4846)
scaleway/docs-content: pages/apple-silicon/how-to/setup-vm-with-utm.mdx

---
meta:

>   title: How to setup a VM with UTM on a hosted Mac mini

Reviewer: ```suggestion
  title: How to set up a VM with UTM on a hosted Mac mini
```

  description: Learn how to create a fully independent and functional virtual machine inside a Scaleway Mac using UTM, a powerful virtualization tool for macOS.
content:

>   h1: How to setup a VM with UTM on a hosted Mac mini

Reviewer: ```suggestion
  h1: How to set up a VM with UTM on a hosted Mac mini
```

  paragraph: Learn how to create a fully independent and functional virtual machine inside a Scaleway Mac using UTM, a powerful virtualization tool for macOS.
tags: vm utm virtual-machine
dates:
  validation: 2025-04-15
  posted: 2025-04-15
categories:
  - bare-metal
---

> 
> 

Reviewer: ```suggestion
```

This guide explains the process to create a fully independent and functional virtual machine inside a Scaleway Mac using UTM, a powerful virtualization tool for macOS.

> UTM is free and open-source software that leverages Apple's Hypervisor framework and QEMU to provide virtualization capabilities.

Reviewer 2: ```suggestion
UTM is a free and open-source software that leverages Apple's Hypervisor framework and QEMU to provide virtualization capabilities.
```

- A [Mac mini](/apple-silicon/how-to/create-mac-mini/)
- [VNC access](/apple-silicon/how-to/access-remote-desktop-mac-mini/) to your Mac

## Installating UTM

1.  Download and install UTM on the Mac mini:
    *   From the official [Mac App Store](https://apps.apple.com/us/app/utm-virtual-machines/id1538878817) or
    *   Directly from the [official website](https://getutm.app/).
2.  Download the macOS IPSW file.
    The IPSW file is a compressed a file format used to install iOS, iPadOS, tvOS, HomePod, watchOS, macOS firmware for devices equipped with Apple silicon.
    *   Obtain the macOS IPSW file from the official [Apple Developer website](https://developer.apple.com/download/).
    <Message type="note">
      You need an [Apple Developer account](/apple-silicon/troubleshooting/cant-create-apple-account/) to access these files.
    </Message>
3.  Create a new virtual machine:
    *   Open UTM and click the "+" button to create a new VM.
    *   Select "Virtualize" and then "macOS".
    *   Browse and select your downloaded IPSW file.
    *   Configure your VM's properties:
        *   Set your desired CPU cores (recommended at least 2 cores).

>         *   Allocate RAM (recommended at least 4GB).

Reviewer: ```suggestion
        *   Allocate RAM (recommended at least 4 GB).
```

            <Lightbox src="scaleway-aps-utm-1.webp" size="medium" alt="The RAM allocation window of UTM with 4096 MB of RAM selected" />

>         *   Configure the disk storage size (recommended at least 64GB).

Reviewer: ```suggestion
        *   Configure the disk storage size (recommended at least 64 GB).
```

>             <Lightbox src="scaleway-aps-utm-2.webp" size="medium" alt="The disk size confgiration window of UTM with 64GB of storage chosen" />

Reviewer: ```suggestion
            <Lightbox src="scaleway-aps-utm-2.webp" size="medium" alt="The disk size configuration window of UTM with 64 GB of storage chosen" />
```

        *   Click "Save" to create the VM.
        <Lightbox src="scaleway-aps-utm-3.webp" size="medium" alt="Summary of the VM configuration displaying the Save button to apply the configuration" />

> 4.  Start and set Up your macOS VM:

Reviewer 3: ```suggestion
4.  Start and set up your macOS VM:
```

    *   Once the VM is installed with your properties, click the "Run" button in UTM.
    *   A GUI window will open displaying your VM.
        <Lightbox src="scaleway-aps-utm-4.webp" size="medium" alt="Mac OS setup wizard" />
    *   Follow the macOS installation process:
        *   Select language preferences.
        *   Set up your Apple ID (optional).

>         *   Create a user account with username and password.

Reviewer: ```suggestion
        *   Create a user account with a username and password.
```

        *   Choose time zone and other basic settings.
        *   Wait for the installation to complete.

> ## Troubleshooting

Reviewer 2: shouldn't this section be its own troubleshooting page?

Author: Not sure if it is worth an own page as it is very specific linked to this guide, 99% of users might never encounter this issue

### You are experiencing IPSW compatibility issues
- Ensure your IPSW file is compatible with UTM by checking the following points:
    - The macOS version is supported by your UTM version
    - The IPSW architecture matches your Mac's architecture (in our case, `arm64` for Apple silicon architecture)
    - The IPSW file is not corrupted (verify checksums if available)
- For performance issues, try adjusting the allocated CPU cores and RAM

## Additional tips
- Take snapshots of your VM before making significant changes
- Use shared folders to easily transfer files between the host and guest systems
- For testing purposes, consider creating multiple VMs with different macOS versions

<Message type="tip">
  For more advanced information about UTM, refer to the [official UTM documentation](https://docs.getutm.app/)
</Message>

> This setup allows you to run a complete and isolated macOS environment within your Scaleway mac instance, perfect for testing applications, MDM profiles, or other macOS-specific tasks.

Reviewer 3: ```suggestion
This setup allows you to run a complete and isolated macOS environment within your Scaleway Mac instance, perfect for testing applications, MDM profiles, or other macOS-specific tasks.
```

> This setup allows you to run a complete and isolated macOS environment within your Scaleway hosted Mac mini, perfect for testing applications, MDM profiles, or other macOS-specific tasks.

Reviewer: ```suggestion
This setup allows you to run a complete and isolated macOS environment within your Scaleway-hosted Mac mini, perfect for testing applications, MDM profiles, or other macOS-specific tasks.
```
